src/main/java/org/jenkinsci/plugins/sma/SMARunner.java

```diff
@@ -58,16 +57,17 @@ public SMARunner(EnvVars jobVariables, String prTargetBranch, SMAJenkinsCIOrgSet
         }
         // Configure using pull request logic
         if (!prTargetBranch.isEmpty() && !shaOverride) {
-            deployAll = false;
-            git = new SMAGit(pathToWorkspace, currentCommit, prTargetBranch, SMAGit.Mode.PRB);
-            previousCommit = git.getPrevCommit();
+//            deployAll = false;
```

Comment:
@aesanch2 I have thought about it a bit and what I wrote earlier make sense but perhaps you could say the same about setting a Pull Request Target branch and it still deploys all. So maybe leave `deployAll = false` enabled. However I do think this exception should be addressed in the documentation where `SMA_DEPLOY_ALL_METADATA` is described. I will modify this PR probably tomorrow and will enable line 60 again.